### PR TITLE
Shawn/buy scanner button

### PR DESCRIPTION
**Pull request by ShawnDhanani**

### CODE REVIEW

File: StockCount/View/OptionViewController.swift [hunk 1]
```diff
@@ -55,6 +60,47 @@ class OptionViewController : UIViewController {
...
+            }
+            
+            if let json = resp.data {
+                do {
+                    let data = try JSON(data: json)
+                    if let link = data["link"].string {
+                        UIApplication.shared.open(URL(string: link)!, options: [:], completionHandler: nil)
+                    }
+                } catch {
+
```
Comment (EricGlaenzer) [inline StockCount/View/OptionViewController.swift]: shoudn't we display an error in the catch?

File: StockCount/View/OptionViewController.swift [hunk 2]
```diff
@@ -55,6 +60,47 @@ class OptionViewController : UIViewController {
     func updateVersion(_ appVer: String, _ captureVer: String) {
         lblVersion.text = "\("version".localized)\(appVer)\n\("Capture".localized) \(captureVer)"
     }
+    
+    @IBAction func onBuyScannerClicked(_ sender: Any) {
+        let regionCode = Locale.current.regionCode
+        
+        if regionCode == "CN" {
```
[the remark this answers is not in the document]
Comment (socketPM) [reply StockCount/View/OptionViewController.swift]: for the CN link, we don't have Socket Store suitable for China. We are using the Amazon link.

[the remark this answers is not in the document]
Comment (EricGlaenzer) [reply StockCount/View/OptionViewController.swift]: I meant a comment in the code...

File: Podfile [hunk 3]
```diff
@@ -7,4 +7,6 @@ target 'StockCount' do
   pod 'Realm', '~>3.13', :modular_headers => true
   pod 'RealmSwift', '~>3.13'
   pod 'SKTCapture', '~>1.1'
+  pod 'Alamofire', '= 4.8.2'
+  pod 'SwiftyJSON'
```
Comment (EricGlaenzer) [inline Podfile]: We should specify a version for `SwiftyJSON`

**Comment by EricGlaenzer** [comment]
This merge request is really similar to the other one. Please update this branch correctly, it should show only the difference you've worked on.

**Comment by ShawnDhanani** [comment]
I had to update barcode consolidation code based on this branch.
Because on this branch, storyboard is updated and I had to update storyboard again for barcode consolidation function.
So, two PR are similar, but barcode consolidation PR is based on this branch, so I think it can merge easily after merge this branch.

File: Podfile (hunk 3 above)
Comment (ShawnDhanani) [reply Podfile]: - will fix on barcode consolidation PR because there are warning issue on those PR that is related this.

**Comment by EricGlaenzer** [comment]
I think then you can close this PR, and use the one that covers everything. There is no point to have 2 almost identical PR, when a fast forward does the trick.

**Comment by ShawnDhanani** [comment]
ok. I'll close this PR, then.

**Pull request closed without merging**